dotnet-desktop-guide/net/wpf/documents/snippets/how-to-display-print-dialog/csharp/MainWindow.xaml.cs

```diff
@@ -0,0 +1,227 @@
+﻿using System;
+using System.IO;
+using System.Reflection;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Documents;
+using System.Windows.Media;
+using System.Windows.Xps.Packaging;
+using Microsoft.Win32;
+
+namespace CodeSampleCsharp
+{
+    /// <summary>
+    /// Interaction logic for MainWindow.xaml
+    /// </summary>
+    public partial class MainWindow : Window
+    {
+        public MainWindow()
+        {
+            InitializeComponent();
+
+            // Events
+            btnBrowse.Click += BtnBrowse_Click;
+            btnPrint.Click += BtnPrint_Click;
```

Comment:
I didn't notice this before, but this is a non-traditional way of hooking up the UI elements to events. These should be declared in the XAML. However, since this article isn't demonstrating XAML, you don't need to change it. Just FYI for the future.

Reply:
Noted, thank you for the tip!